### DISCUSSION TITLE
How to retrieve the materialization result of an asset inside an asset_sensor

### DISCUSSION THREAD

**Discussion by Zibc** [dagster-io/dagster#20106] [Q&A]
Hello all,

I've been discovering dagster since some weeks and started trying to use it for a few tasks.

In my current pipeline, I have two asset graphs that are "connected" via an asset_sensor.
When the first graph is materialized, the second is kicked-off for materialization.
The last asset of the first graph return some file paths that I would like to retrieve from with the asset_sensor.
I've seen this documentation page: https://docs.dagster.io/concepts/ops-jobs-graphs/graphs#loading-an-asset-as-an-input
but I have no idea how to apply this from within the asset_sensor ...

Any hints ?

**Comment by sbusso** [comment]
I was also looking at the same in the documentation maze. I found this issue and adapted it: https://github.com/dagster-io/dagster/discussions/14805. It seems to be doing the job, but I'm not sure it is the right way.

```python
defs = Definitions(assets=[asset1])

@asset_sensor(
asset_key=AssetKey("asset1"),
job=a_job,
)
def my_sensor():
	value = defs.load_asset_value("asset1")
```
